index.js

```diff
@@ -0,0 +1,33 @@
+const validateLineup = (lineup) => {
+  for (let i = 0; i < lineup.length; i++) {
+    let count = lineup[i]
+    const salaries = lineup.reduce((totalSum, lineup) => {
+      return lineup.salary + totalSum
+    }, 0)
+    let validSalary = true
+    let validPosition = true
+    let validTeam = true
+    let validGame = true
+    let completeLineup = true
+
+    if (count.position['OF'] > 3 && count.position['P', 'C', '1B', '2B', 'SS'] !== 1) {
```

Comment:
goal of these rules was to **count** how many players (in the lineup) have the position 'OF' . and if exactly 3, then good and return 3. else bad and return false

you need to check how many times a the position OF occurs in the lineup. not if the value of position 'OF' is more than 2. 

here 'position' property has value 'OF' in the player and 'OF' is not a property with a number on it. same with other positions of players. please understand the player/lineup object in more detail and also i recomend asking if you understand the problem correctly